lib/snov.rb

```diff
@@ -33,6 +33,7 @@ def self.use_fake?
 require 'snov/in_memory_token_storage'
 require 'snov/redis_token_storage'
 require 'snov/fake_client'
+require 'camel_snake_struct'
```

Comment:
Question: what is the purpose of using `camel_snake_struct`? what does this repo `snov-ruby` do?